static/app/utils/withSavedSearches.tsx

```diff
@@ -16,10 +19,37 @@ type State = {
   savedSearches: SavedSearch[];
 };
 
+/**
+ * HOC to provide saved search data class components.
```

Comment:
nit: should this be "... saved search data _to_ class components..."

Reply:
it definitely should!